internal/fleet/agent_policy_resource_test.go

```diff
@@ -111,8 +111,8 @@ resource "elasticstack_fleet_agent_policy" "test_policy" {
   name        = "%s"
   namespace   = "default"
   description = "This policy was updated"
-  monitor_logs = true
-  monitor_metrics = true
+  monitor_logs = false
+  monitor_metrics = false
```

Comment:
Is it worth having these with different values (i.e `logs = true, metrics = false`) to better test the parsing of each type?

Reply:
I think that's a great idea, it would act as a good regression test for parsing each type.